pkgs/development/tools/build-managers/sbt/scala-native.nix

```diff
@@ -1,13 +1,9 @@
 { lib, sbt, makeWrapper, boehmgc, libunwind, re2, llvmPackages, zlib }:
 
-sbt.overrideDerivation(old: {
-  nativeBuildInputs = [ makeWrapper ];
+sbt.overrideAttrs(previousAttrs: {
+  nativeBuildInputs = (previousAttrs.nativeBuildInputs or [ ]) ++ [ makeWrapper ];
 
-  version = "0.13.16";
-
-  sha256 = "033nvklclvbirhpsiy28d3ccmbm26zcs9vb7j8jndsc1ln09awi2";
-
-  postFixup = ''
```

Comment:
@Mic92 fixes the no-op you noticed in https://github.com/NixOS/nixpkgs/pull/29148#issuecomment-328784473
after switching to `overrideAttrs` `finalAttrs.version` is used in sbt so now it actually does something and breaks